### DISCUSSION TITLE
Can we extract Start Reaction times from available data?

### DISCUSSION THREAD

**Discussion by harish5p** [theOehrly/Fast-F1#177] [General]
AWS uses several sensors to predict the reaction times documented here: https://aws.amazon.com/f1/start-analysis/

I read in [this discussion thread](https://github.com/theOehrly/Fast-F1/discussions/127) we have data available just before the start. Do you think we can extract start reaction times? @theOehrly

**Comment by theOehrly** [comment]
The data we have has a too low resolution to do this. The accuracy of the result would be very low and not usable. Consider that the sample time of the raw data is 200-250 ms.
